stopes/ui/seamlisten/backend/app/fileviewer.py

```diff
@@ -161,3 +169,24 @@ async def general_query(query: DefaultQuery) -> Response:
         a line with audio file, start and end timestamps
         """,
         )
+
+
+import os
+
+
+def gather_folder_contents(folder_path, max_depth=3):
+    folder_contents = []
+    for root, dirs, files in os.walk(folder_path, topdown=True):
+        depth = root[len(folder_path) + len(os.path.sep) :].count(os.path.sep)
```

Comment:
again, use the Path API to do this kind of things

Reply:
done